src/Helper/SerializationHelper.php

```diff
@@ -142,8 +142,22 @@ public static function getProperties($obj)
             $rc = new ReflectionClass($obj);
             do {
                 $rp = array();
+
+                // Note that we will not include any properties in traits
+                $traits = $rc->getTraits();
+                $skipProperties = array();
+                if (!empty($traits)) {
+                    foreach ($traits as $trait) {
+                        foreach ($trait->getProperties() as $traitProperty) {
+                            $skipProperties[] = $traitProperty->getName();
```

Comment:
Can the trait have a property that doesn't have a `getName()` method?

Reply:
Nopes, `getProperties` returns an array of `ReflectionProperty` (http://php.net/manual/en/class.reflectionproperty.php) and `ReflectionProperty` always has method `getName`